File: app/src/main/java/com/example/karumbi/moviedb/model/Genre.java
package com.example.karumbi.moviedb.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by Karumbi on 11/10/2017.
 */

public class Genre {
  @SerializedName("id")
  @Expose
  private int id;
  @SerializedName("name")
  @Expose
  private String name;

  public int getId() {
    return id;
  }

  public void setId(int id) {
    this.id = id;
  }

  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }
}

File: app/src/main/java/com/example/karumbi/moviedb/model/Genre.kt
package com.example.karumbi.moviedb.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by Karumbi on 11/10/2017.
 */

class Genre {
  @SerializedName("id")
  @Expose
  var id: Int = 0
  @SerializedName("name")
  @Expose
  var name: String? = null
}
